src/components/Button/Button.tsx

```diff
@@ -69,7 +69,8 @@ export class Button extends React.Component<IButtonProps, IButtonState> {
           'aria-label': ariaLabel,
           'aria-labelledby': ariaLabel ? null : labelId,
           'aria-describedby': ariaDescription ? ariaDescriptionId : description ? descriptionId : null,
-          'ref': (c: HTMLButtonElement): HTMLButtonElement => this._buttonElement = c
+          'ref': (c: HTMLButtonElement): HTMLButtonElement => this._buttonElement = c,
+          'href' : href ? href : null
```

Comment:
This doesn't seem to be causing a warning during gulp serve, nor in console on navigation to the page.

I've made the change requested none-the-less.